package.json

```diff
@@ -35,7 +35,7 @@
     "dependencies": {
         "array.prototype.every": "^1.0.0",
         "array.prototype.some": "^1.0.0",
-        "async": "^2.6.2",
+        "async": "^2.6.3",
```

Comment:
this is unnecessary - semver means it comes in by default; unless you have a lockfile, in which case it’s up to you to update it (and npm audit fix can handle it for you)